wasm-wrappers/src/lib.rs

```diff
@@ -547,6 +594,61 @@ pub fn encode_output_issue_fungible_token(
     Ok(output.encode())
 }
 
+/// Given the parameters needed to issue an NFT, and a network type (mainnet, testnet, etc),
+/// this function creates an output that issues that NFT.
+#[allow(clippy::too_many_arguments)]
+#[wasm_bindgen]
+pub fn encode_output_issue_nft(
+    token_id: &str,
+    authority: &str,
+    name: &[u8],
+    ticker: &[u8],
+    description: &[u8],
+    media_hash: &[u8],
+    creator: Option<String>,
+    media_uri: Option<Vec<u8>>,
+    icon_uri: Option<Vec<u8>>,
+    additional_metadata_uri: Option<Vec<u8>>,
+    network: Network,
+) -> Result<Vec<u8>, Error> {
+    let chain_config = Builder::new(network.into()).build();
+    let token_id = parse_addressable(&chain_config, token_id)?;
+    let authority = parse_addressable(&chain_config, authority)?;
+    let name = name.into();
+    let ticker = ticker.into();
+    let media_uri = media_uri.into();
+    let icon_uri = icon_uri.into();
+    let media_hash = media_hash.into();
```

Comment:
These and other fields need to be tested before approving the output. See in check_transaction how we do that. Feel free to factor out things that are needed to just test an output.